src/main/java/seedu/duke/commons/util/JsonUtil.java

```diff
@@ -11,11 +11,14 @@
 import java.io.Reader;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 public class JsonUtil {
```

Comment:
The `JsonUtil` class is meant to be a common utility class that can be used by many other classes. Currently, the functionality is too module-specific. [Reference](https://github.com/se-edu/addressbook-level3/blob/master/src/main/java/seedu/address/commons/util/JsonUtil.java).

Consider extracting some statements into utility methods (e.g. `readJsonFIle`) inside this class. Some module-related methods can be refactored to the `FullModuleList` or `ModuleList` class (whichever is more suitable).